<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Explorer</class>
 <widget class="QDialog" name="Explorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Explorer</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_0">
    <attribute name="title">
     <string>实时信息</string>
    </attribute>
    <widget class="QGroupBox" name="cpuGroupBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>351</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>CPU</string>
     </property>
     <widget class="QProgressBar" name="cpuProgressBar">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="cpuLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>使用率：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="memoryGroupBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>351</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>内存</string>
     </property>
     <widget class="QLabel" name="memoryLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>利用率：</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="memoryProgressBar">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="numberMemLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>进程信息</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>11</y>
       <width>421</width>
       <height>301</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PPID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>线程数</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>基本信息</string>
    </attribute>
    <widget class="QGroupBox" name="sysGroupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>411</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>系统</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>391</width>
        <height>112</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="sysFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="showHostNameLabel">
         <property name="text">
          <string>主机名：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="showVersionLabel">
         <property name="text">
          <string>系统版本：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="showCpuLabel">
         <property name="text">
          <string>CPU：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="showClassCpuLabe">
         <property name="text">
          <string>CPU类型：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="sHostNameLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="sVersionLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="sCpuLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="sClassCpuLabe">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="timeGroupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>411</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>时间</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>391</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="timeFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="showBeginLabel">
         <property name="text">
          <string>系统启动时间：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="showRunLabel">
         <property name="text">
          <string>系统运行时间：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="sBeginLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="sRunLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="killPushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>结束进程</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
